modules/identity-provider-keystone-about.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-keystone-identity-provider.adoc
+
+[id="identity-provider-keystone-about_{context}"]
+= About Keystone authentication
+
+link:http://docs.openstack.org/developer/keystone/[Keystone] is an OpenStack project that provides identity, token, catalog, and policy services.
+
+You can configure the integration with Keystone so that the new {product-title} users are based on either the Keystone user names or unique Keystone IDs. With both methods, users log in by entering their Keystone user name and password. Basing the {product-title} users on of the Keystone ID is more secure because if you delete a Keystone user and create a new Keystone user with that user name, the new user might have access to the old user's resources.
```

Comment:
Unneeded `of`?
```suggestion
You can configure the integration with Keystone so that the new {product-title} users are based on either the Keystone user names or unique Keystone IDs. With both methods, users log in by entering their Keystone user name and password. Basing the {product-title} users on the Keystone ID is more secure because if you delete a Keystone user and create a new Keystone user with that user name, the new user might have access to the old user's resources.
```

Reply:
good catch, removed!